gravitee-apim-console-webui/src/management/api/endpoints-v4/endpoint-group/create/api-endpoint-group-create.component.ts

```diff
@@ -0,0 +1,166 @@
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, HostBinding, Inject, OnInit, ViewChild } from '@angular/core';
+import { STEPPER_GLOBAL_OPTIONS } from '@angular/cdk/stepper';
+import { of, Subject } from 'rxjs';
+import { GIO_FORM_FOCUS_INVALID_IGNORE_SELECTOR, GioJsonSchema } from '@gravitee/ui-particles-angular';
+import { MatStepper } from '@angular/material/stepper';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { distinctUntilChanged, switchMap, takeUntil } from 'rxjs/operators';
+import { StateService } from '@uirouter/angular';
+
+import { ApiEndpointGroupSelectionComponent } from '../selection/api-endpoint-group-selection.component';
+import { ApiEndpointGroupConfigurationComponent } from '../configuration/api-endpoint-group-configuration.component';
+import { ApiEndpointGroupGeneralComponent } from '../general/api-endpoint-group-general.component';
+import { ApiV2Service } from '../../../../../services-ngx/api-v2.service';
+import { isUnique } from '../../../../../shared/utils';
+import { ApiV4, EndpointGroupV4, EndpointV4Default, UpdateApiV4 } from '../../../../../entities/management-api-v2';
+import { UIRouterState, UIRouterStateParams } from '../../../../../ajs-upgraded-providers';
+import { SnackBarService } from '../../../../../services-ngx/snack-bar.service';
+import { ConnectorPluginsV2Service } from '../../../../../services-ngx/connector-plugins-v2.service';
+
+@Component({
+  selector: 'api-endpoints-group-create',
+  template: require('./api-endpoint-group-create.component.html'),
+  styles: [require('./api-endpoint-group-create.component.scss')],
+  providers: [
+    {
+      provide: STEPPER_GLOBAL_OPTIONS,
+      useValue: { displayDefaultIndicatorType: false, showError: true },
+    },
+  ],
+})
+export class ApiEndpointGroupCreateComponent implements OnInit {
+  private unsubscribe$: Subject<boolean> = new Subject<boolean>();
+
+  @HostBinding(`attr.${GIO_FORM_FOCUS_INVALID_IGNORE_SELECTOR}`)
+  private gioFormFocusInvalidIgnore = true;
+
+  @ViewChild(ApiEndpointGroupSelectionComponent)
+  private apiEndpointGroupsSelectionComponent: ApiEndpointGroupSelectionComponent;
+
+  @ViewChild(ApiEndpointGroupGeneralComponent)
+  private apiEndpointGroupGeneralComponent: ApiEndpointGroupGeneralComponent;
+
+  @ViewChild(ApiEndpointGroupConfigurationComponent)
+  private apiEndpointGroupConfigurationComponent: ApiEndpointGroupConfigurationComponent;
+
+  @ViewChild('stepper', { static: true })
+  private matStepper: MatStepper;
```

Comment:
is it used or can we remove those line ?

Reply:
Just tested it, and it is **not** necessary 👍